src/screens/Athena.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useUserContext } from '../global/user-context'
+import Onboarding from './onboarding/Onboarding'
+import Navbar from './navbar/Navbar'
+
+const Athena = () => {
+
+    const { state } = useUserContext()
+
+    return (
+        <>
+            { state.user ? <Navbar /> : <Onboarding />}
```

Comment:
Careful with this line. I believe state.user is always defined because of the default value of state.user is an empty object literal, {}. I would check for state.user.id instead